apps/api/src/server/repositories/document.repository.js

```diff
@@ -88,10 +116,11 @@ export const updateDocumentStatus = ({ guid, status }) => {
  * Returns total number of documents in a folder on a case
  *
  * @param {number} folderId
+ * @param {boolean} isDeleted
  * @returns {import('@prisma/client').PrismaPromise<number>}
  */
-export const getDocumentsCountInFolder = (folderId) => {
+export const getDocumentsCountInFolder = (folderId, isDeleted = false) => {
 	return databaseConnector.document.count({
-		where: { folderId }
+		where: { folderId, isDeleted }
```

Comment:
What value might you supply if you wanted to say "count all documents, no matter if it's deleted"?